packages/eas-cli/src/analytics.ts

```diff
@@ -32,11 +34,33 @@ export async function initAsync(): Promise<void> {
   }
   const amplitudeEnabled = await UserSettings.getAsync('amplitudeEnabled', true);
   if (amplitudeEnabled) {
-    const apiKey =
+    const config =
       process.env.EXPO_STAGING || process.env.EXPO_LOCAL
-        ? 'cdebbc678931403439486c4750781544'
-        : '4ac443afd5073c0df6169291db1d3495';
-    client = Amplitude.init(apiKey, { retryClass: new Amplitude.OfflineRetryHandler(apiKey) });
+        ? {
+            // dev environment
+            amplitudeWriteKey: 'cdebbc678931403439486c4750781544',
+            rudderstackWriteKey: '1wpX20Da4ltFGSXbPFYUL00Chb7',
+            rudderstackDataPlaneURL: 'https://cdp.expo.dev',
```

Comment:
if there is staging env this url should point to it, if not we probably do not want to send those events at all

Reply:
There is no staging URL for RudderStack, but I'd like to send development events to:

- match how other codebases are set up
- enable testing of analytics pipelines and tasks related to analytics events